bin/ext/beeline.sh

```diff
@@ -29,7 +29,7 @@ beeline () {
     hadoopClasspath="${HADOOP_CLASSPATH}:"
   fi
   export HADOOP_CLASSPATH="${hadoopClasspath}${HIVE_CONF_DIR}:${beelineJarPath}:${superCsvJarPath}:${jlineJarPath}"
-  export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Dlog4j.configurationFile=beeline-log4j2.properties "
+  export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Dlog4j.configurationFile=beeline-log4j2.properties --add-opens=java.base/java.net=ALL-UNNAMED "
```

Comment:
If users want to deploy the HS2/HMS against JDK17, in which script files should they add the jdk17 parameter?
Should we change this files as well?
https://github.com/apache/hive/blob/master/bin/ext/hiveserver2.sh
https://github.com/apache/hive/blob/master/bin/ext/metastore.sh